packages/extension/src/companion/CompanionPopupButton.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, ReactElement, useContext, useRef } from 'react';
+import classNames from 'classnames';
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import SimpleTooltip from '@dailydotdev/shared/src/components/tooltips/SimpleTooltip';
+import CompanionFilledIcon from '@dailydotdev/shared/icons/filled/companion.svg';
+import CompanionOutlineIcon from '@dailydotdev/shared/icons/outline/companion.svg';
+import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import { CompanionPermission } from './CompanionPermission';
+import { useExtensionPermission } from './useExtensionPermission';
+
+export const COMPANION_ALERTS_LOCAL_KEY = 'companion:alerts';
+
+interface CompanionAlerts {
+  displayCompanionPopup?: boolean;
+}
+
+export const getCompanionAlerts = (): CompanionAlerts => {
+  const data = window.localStorage.getItem(COMPANION_ALERTS_LOCAL_KEY);
+
+  if (!data) {
+    return {};
+  }
+
+  try {
+    return JSON.parse(data);
+  } catch (ex) {
+    return {};
+  }
+};
+
+export const updateCompanionAlerts = (
+  current: CompanionAlerts,
+  updated: CompanionAlerts,
+): CompanionAlerts => {
+  const data = { ...current, ...updated };
+  window.localStorage.setItem(COMPANION_ALERTS_LOCAL_KEY, JSON.stringify(data));
```

Comment:
Should we not opt to use `storageWrapper` for this?

Reply:
Good point, yes, we should use it here.